### DISCUSSION TITLE
Can't download YouTube video

### DISCUSSION THREAD

**Discussion by theophasbie** [norkunas/youtube-dl-php#195] [Q&A]
Hi, I need help to fix this issue    

public function download() {

        $yt = new YoutubeDl();
        $yt->setBinPath(ENV('YOUTUBE_DL'));
 
        $collection = $yt->download(
            Options::create()
                ->downloadPath('C:\Users\theop\Downloads')
                ->url('https://www.youtube.com/watch?v=Cag-pf1Iemw')
        );

        dd($collection);

        foreach ($collection->getVideos() as $video) {

            if ($video->getError() !== null) {
                echo "Error downloading video: {$video->getError()}.";
            } else {
                echo $video->getTitle(); // Will return Phonebloks

            }
        }
    }

I get a blank page without any error

This is my ENV variable YOUTUBE_DL=C:\youtube-dl\youtube-dl.exe

P.S : I'm able to download YouTube video using my terminal means youtube-dl work

**Comment by norkunas** [comment]
Hi, then check php logs what they say :) or enable some error handler to get some errors :)

**Reply by theophasbie** [reply]
Hi. Thanks for your reply. Yes I have deleted the content of php_errors.log, laravel.log and then rexecute the script. Nothing to report in both files.

**Reply by theophasbie** [reply]
I have also put APP_DEBUG to true in my env

**Reply by norkunas** [reply]
Did you find the cause?
